packages/core/src/services/ai/estimateCost/xai.ts

```diff
@@ -0,0 +1,19 @@
+import { createModelSpec } from './helpers'
+
+export const XAI_MODELS = createModelSpec({
```

Comment:
what do you think about `createModelSpec`? es comodo?

Reply:
yup, it's good!